contracts/usul/BaseTokenVoting.sol

```diff
@@ -0,0 +1,172 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts-upgradeable/utils/cryptography/draft-EIP712Upgradeable.sol";
+import "@openzeppelin/contracts/utils/cryptography/ECDSA.sol";
+import "./BaseStrategy.sol";
+
+/// @title An abstract contract used as a base for ERC-20 token voting strategies
+abstract contract BaseTokenVoting is BaseStrategy {
+    enum VoteType {
+        Against,
+        For,
+        Abstain
```

Comment:
We should capitalize this one too

Reply:
Also maybe we should change it to YES, NO, ABSTAIN, since they're referred to `yesVotes` / `noVotes` below, rather than for/against, or alternatively change those to `forVotes` / `againstVotes`

Reply:
Yeah sounds good. I like Yes / No more than For / Against

Reply:
Done